src/rpc.rs

```diff
@@ -12,80 +12,125 @@
  ********************************************************************************/
 use crate::UPClientZenoh;
 use async_trait::async_trait;
-use std::{string::ToString, time::Duration};
+use std::{string::ToString, sync::Arc, time::Duration};
 use tracing::error;
-use up_rust::{RpcClient, RpcClientResult, UAttributesError, UMessage, UMessageError, UUri};
+use up_rust::{
+    communication::{CallOptions, RpcClient, ServiceInvocationError, UPayload},
+    LocalUriProvider, UAttributes, UCode, UMessageType, UPayloadFormat, UPriority, UStatus, UUri,
+};
 use zenoh::prelude::r#async::*;
 
+pub struct ZenohRpcClient {
+    transport: Arc<UPClientZenoh>,
```

Comment:
Given that the `RpcClient` is implemented on a different struct `ZenohRpcClient` now, I think it's safe to rename `UPClientZenoh` to `UPTransportZenoh` since it truly only handles the uP-L1 uTransport APIs now.

Reply:
Agree with you